CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## Please, call me `BlobServiceClient` - `BlockBlobService` was my father's name <Badge text="beta" type="success" />
```

Comment:
We normally just have the number name here, the text name goes in the github releases only.